### PR TITLE
layerscape: update kernel patches and packages to NXP LSDK1712

### PR DESCRIPTION
**Pull request by yangbolu1991**
These are the patches from NXP Layerscape OpenWrt 1802 release. I attached the release notes and release announcement here. Please help to reveiw these patches.

Thanks a lot.

[Release announcement.docx](https://github.com/openwrt/openwrt/files/1698396/Release.announcement.docx)
[Release notes.txt](https://github.com/openwrt/openwrt/files/1698397/Release.notes.txt)

### CODE REVIEW

File: package/network/utils/layerscape/fmc/Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+PKG_SOURCE_VERSION:=8c9f127138dc803bf58bef75e9fc092dc60f9c53
+PKG_MIRROR_HASH:=199b91c0a49ab4a4aa1eded7703ea1c11241e22d21a41f00fe992c043b28b467
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/fmc
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=@TARGET_layerscape +libxml2 +tclap +fmlib +libstdcpp
+  TITLE:=Layerscape Frame Manager Configuration Tool
```
Comment (blogic) [inline package/network/utils/layerscape/fmc/Makefile]: build test fails as libxml2 is not part of the base feed. shall we drop the fmc stuff from the PR for now and merge the rest ?

**Comment by blogic** [comment]
merged, leaving out the fmc stuff.

File: package/network/utils/layerscape/fmc/Makefile (hunk 1 above)
Comment (yangbolu1991) [reply package/network/utils/layerscape/fmc/Makefile]: @blogic 
Sorry for my late response. I just come back to office from Chinese New Year holidays today.
And thanks a lot for your reveiwing.

So what should I do, if I want to add fmc support? Move libxml to base feed?
Thanks.

Comment (yangbolu1991) [reply package/network/utils/layerscape/fmc/Makefile]: Any suggestion? Actually fmc is an important tool for customer :)

Comment (yangbolu1991) [reply package/network/utils/layerscape/fmc/Makefile]: @hauke 
Also add Hauke for some suggestion :)

Hi Hauke,
Any idea for this problem? Thanks a lot.

**Pull request closed without merging**